Task3/EEL_task_3/src/main/java/com/epam/esm/giftcertificate/repository/GiftCertificateRepositoryImpl.java

```diff
@@ -0,0 +1,97 @@
+package com.epam.esm.giftcertificate.repository;
+
+import com.epam.esm.giftcertificate.model.GiftCertificate;
+import com.epam.esm.tag.model.Tag;
+import com.epam.esm.utils.AppQuery;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Repository
+public class GiftCertificateRepositoryImpl implements GiftCertificateRepository {
+    private final JdbcTemplate jdbcTemplate;
+
+    @Autowired
```

Comment:
@Autowired annotation isn't needed when you have 1 constructor as this constructor would be used be default